Model/simulation.py

```diff
@@ -1015,7 +1009,7 @@ def plot_fee_estimate(
     end_block = 680000
 
     sim.run(start_block, end_block)
-    # sim.plot(show=True)
+    (rep, df) = sim.plot(show=False)
```

Comment:
?

Reply:
Needed to output the "report" as dataframe too so I could handle results programatically for plotting and analysis.

Reply:
But i mean you don't use these vars